css/style.css

```diff
@@ -11,4 +11,92 @@ font-family: 'Raleway', sans-serif;
 
 *{
     font-family: 'Raleway', sans-serif;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
```

Comment:
add a useful comment to  tell us what is that code and the use of it ( to which section also )